apps/sports-helsinki/src/hooks/useHandleEventsOrderChange.ts

```diff
@@ -0,0 +1,20 @@
+import { replaceQueryParamValuesInRouter } from 'events-helsinki-components';
+import type { Option } from 'events-helsinki-components';
+import { useRouter } from 'next/router';
+import { useCallback } from 'react';
+
+const useHandleEventsOrderChange = () => {
```

Comment:
I would probably just rename this to `useReplaceQueryParamValuesInRouter` and move it to components code. If `replaceQueryParamValuesInRouter` is not used anywhere else than hooks then it doesnt need to have separate function. That's mostly an opinion for cleaner code, not something that needs to be fixed necessary :)

Reply:
Renamed to `useSetSortQueryParamToOptionValue` and made it like this:
```typescript
const useSetSortQueryParamToOptionValue = () => {
  const router = useRouter();
  return useCallback(
    async (option: Option) => {
      return router.push(
        {
          pathname: router.pathname,
          query: { ...router.query, sort: option.value },
        },
        undefined,
        { shallow: true }
      );
    },
    [router.asPath, router.basePath, router.locale]
  );
};
```